docs/api.rst

```diff
@@ -1,48 +0,0 @@
-..
-    Copyright 2017 - Swiss Data Science Center (SDSC)
-    A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
-    Eidgenössische Technische Hochschule Zürich (ETHZ).
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
-
-Low-level API
-=============
-
-This API is built on top of REST API endpoints exposed by Renga services.
-
-.. warning::
-
-   Renga services are currently in **beta preview** status and they are
-   subject to change in forseenable future.
-
-.. automodule:: renga.api
-   :members:
-
-Projects
```

Comment:
Here we should include new project model, CWL, datasets, ...

Reply:
that's true and that's the plan -- however, the full reworking of docs is a bigger task that I was going to leave until later. I just wanted to clean up and have a minimal set of stuff necessary in the repo because we have a ton of things left over from the previous iteration. But I can keep it in here as a skeleton if you insist.